tests/mutation_test.cc

```diff
@@ -1320,6 +1320,99 @@ SEASTAR_THREAD_TEST_CASE(test_mutation_upgrade_type_change) {
     assert_that(m).is_equal_to(m2);
 }
 
+SEASTAR_THREAD_TEST_CASE(test_row_marker) {
+    auto ttl = std::chrono::seconds(1);
```

Comment:
After `using std::chrono_literals` you can just write `1s`.

Reply:
Done.